pcsx2/Frontend/ImGuiManager.cpp

```diff
@@ -521,25 +522,48 @@ static void DrawPerformanceOverlay()
 	constexpr bool paused = false;
 #endif
 
+#ifdef PCSX2_CORE
+	// TODO - is there a way to forcibly update the imgui frame without requiring to wait until the next vsync?
```

Comment:
`GetMTGS().PresentCurrentFrame()`, but if you want to call that from anywhere other than the CPU thread, you'll need to wrap it in `EmuThread` Qt-side.